### PR TITLE
Update license file in copyright headers

### PR DESCRIPTION
**Pull request by tamiko**
In reference to #6706
In reference to #6716

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ## it, and/or modify it under the terms of the GNU Lesser General
 ## Public License as published by the Free Software Foundation; either
 ## version 2.1 of the License, or (at your option) any later version.
-## The full text of the license can be found in the file LICENSE at
+## The full text of the license can be found in the file LICENSE.md at
 ## the top level of the deal.II distribution.
```
Comment (tamiko) [inline CMakeLists.txt]: While we are at it, shall we also rename "deal.II distribution" to something like "deal.II repository"?

**Comment by tamiko** [comment]
@tjhei Here you go.

File: CMakeLists.txt (hunk 1 above)
Comment (bangerth) [reply CMakeLists.txt]: Maybe "the top level directory of deal.II"?

**Comment by bangerth** [comment]
I'm ok with the change as is, but also happy to wait if you wanted to fix the one piece of text.

File: CMakeLists.txt (hunk 1 above)
Comment (tamiko) [reply CMakeLists.txt]: I like that. Let me do that change.

**Comment by tamiko** [comment]
@bangerth Updated.

**Comment by masterleinad** [comment]
Running one tester should be sufficient for this.

**Pull request merged**